content/blog/2022/platform-tools.md

```diff
@@ -0,0 +1,497 @@
+Bram points out
+that authors are not taking full advantage
+of variable fonts --
+we're mostly changing the font weights,
+and selecting from standard values.
+
+As someone who has spent hours
+searching through lists of variable fonts,
+that doesn't surprise me much.
+Most variable fonts are either experimental display fonts
+without a lot of practical use,
+or they only provide a single weight axis.
+While I love having all my weights in a single file
+and smoothly transitioning from one weight to another,
+_specialty weights_ are not a high priority for me.
+I'll start using more powerful font variables
+once there are more fonts available
+with more interesting variables to manipulate.
+
+Most of us are also using
+the over-complicated `font-feature-settings` property,
+instead of the simpler `font-variant`.
+I've been doing that on my site as well!
+
+Why did I think that was the best approach?
+Likely because that's the generated output
+recommended by
+[Wakamai Fondue](https://wakamaifondue.com/) --
+a popular tool for inspecting font capabilities.
+It's a great tool,
+filling a much-needed gap,
+but I wonder how the usage stats would change
+if they updated their output CSS?
+
+Maybe even more common,
+much of the web is built using CSS libraries
+like Bootstrap,
+or third-party themes
+provided by a site-builder or CMS.
+The influence of both
+WordPress and Bootstrap is clear
+throughout the Almanac data.
+When Rachel notes the popularity of
+flexbox (very popular) vs grid (not so much),
+I wonder how much of that
+can be traced back to
+Bootstrap 'grids'
+[being built in flexbox](https://getbootstrap.com/docs/5.2/layout/grid/).
+
+I don't think that's the only factor.
+It's possible authors have settled on flexbox as _good-enough_,
+and they don't think it's worth their time
+learning another technique.
+I can understand that,
+but it's unfortunate.
+I think people are missing out
+on one of the coolest features
+added to CSS _ever_.
+Oh well.
+
+## Tools update faster? Right?
+
+When we talk about web trends,
+I often hear people refer back to
+the idea of 'pace layers' on the web:
+
+{{ embed.figure(
+  data=adactio.data,
+  caption=adactio.caption
+) }}
+
+In context,
+Jeremy is not just referring to
+the web languages themselves --
+JS isn't necessarily changing faster than CSS --
+but also the _ecosystems_ of these languages.
+The closer we get to the core functionality of the web,
+the slower each language/ecosystem is likely to change.
+Even within a given language,
+we expect our tools and conventions (_fashion_)
+to move faster than the languages themselves.
+
+But that can only happen
+if the tools are designed to
+sit on top of their underlying languages.
+Once the tools stand _between_ us and the language,
+we become entirely reliant on tool-builders
+to determine what features are available.
+
+When our primary tools
+intervene so strongly
+between us and our 'materials' --
+the core web languages --
```

Comment:
```suggestion
the core web languages ---
```

Reply:
I'm curious about these -- I don't think we use `---` elsewhere on the site for anything other than quote attributions?

Reply:
Ah, you're right. The styleguide specifically forbirds em dashes for asides. I'll remove these suggestions!